Library/Homebrew/cmd/update-report.rb

```diff
@@ -104,6 +105,36 @@ def update_report
       next unless tap.git?
       next if tap.core_tap? && ENV["HOMEBREW_INSTALL_FROM_API"].present? && args.preinstall?
 
+      # HOMEBREW_CORE_MERGE_MAINTAINER can be set by the maintainers
+      # who merge homebrew-core into linuxbrew-core to avoid being migrated
+      # We only migrate HOMEBREW_DEVELOPER users right now
+      # Don't migrate on CI either
+      if ENV["HOMEBREW_DEVELOPER"].present? &&
+         ENV["HOMEBREW_CORE_MERGE_MAINTAINER"].blank? &&
+         ENV["GITHUB_ACTIONS"].blank? &&
+         tap.core_tap? &&
+         tap.remote == "https://github.com/Homebrew/linuxbrew-core"
+        puts_stdout_or_stderr "Migrate from linuxbrew-core to homebrew-core" unless args.quiet?
+
+        LINUXBREW_CORE_MIGRATION_LIST.each do |name|
+          formula = Formula[name]
+          next unless formula.any_version_installed?
+
+          selected_keg = nil
+          version = formula.installed_kegs.map(&:version).max
+          formula.installed_kegs.each do |keg|
+            selected_keg = keg if version == keg.version
+          end
+          tab = Tab.for_keg(selected_keg)
+          tab.version_scheme = tab.version_scheme - 1
```

Comment:
```suggestion
          tab.version_scheme = -1
```
unless that doesn't work

Reply:
It does. I thought you wanted to use the current tab.version_scheme and then remove 1, not directly set it to -1. I think the result would be the same in the end.

I checked and I think the list of impacted formulae does not include a formula with a specific version_scheme, so the result will be -1 for them anyway, for both techniques. So I am going to use -1 directly.

Reply:
> I checked and I think the list of impacted formulae does not include a formula with a specific version_scheme, so the result will be -1 for them anyway, for both techniques. So I am going to use -1 directly.

👍🏻 thanks!